api/representation.go

```diff
@@ -33,7 +33,7 @@ func (tr taskRequest) ToTaskConfig() (config.TaskConfig, error) {
 	// Convert source input
 	if tr.ModuleInput != nil {
 		if tr.ModuleInput.Services != nil {
-			si := &config.ServicesSourceInputConfig{
+			si := &config.ServicesModuleInputConfig{
```

Comment:
Caught this in another review, consider renaming `si` instances to `mi` or something else that better reflects `ModuleInput` rather than `SourceInput`

Reply:
Sure, I actually thought the "s" in "si" stood for "services". I ended up going with "input" since it was most readable for me.